modules/apps/frontend-js/frontend-js-sample-web/.npmbundlerrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This should go in `npmscripts.config.js`, since we removed all `.npmbundlerrc` files long ago, moving their contents there.

Reply:
Thanks! moved the contents to npmscripts.config.js